lib/src/progress-bar/ProgressBar.tsx

```diff
@@ -46,7 +68,7 @@ const DxcProgressBar = ({
   );
 };
 
-const BackgroundProgressBar = styled.div`
+const BackgroundProgressBar = styled.div<ProgressBarPropsType>`
```

Comment:
You can simplify the styles. Example:

Before:
```
  background-color: ${(props) => (props.overlay === true ? `${props.theme.overlayColor}` : "transparent")};
  opacity: ${(props) => props.overlay === true && "0.8"};
  width: ${(props) => (props.overlay === true ? "100%" : "")};
```

After:
```
  ${({ overlay, theme }) =>
    overlay
      ? `background-color: ${theme.overlayColor};
          opacity: 0.8";
          width: 100%";`
      : `background-color: "transparent";`}
```

Reply:
Maybe to leave it as it is? Because there are more styles with that problem so I think in the end it will be less readable than the actual solution.
I mean, in the end what you propose it will be like this (including the styles that are left):
```
  ${({ overlay, theme }) =>
    overlay
      ? `background-color: ${theme.overlayColor};
        opacity: 0.8;
        width: 100%;
        justify-content: center;
        height: 100vh;
        align-items: center;
        max-width: 100%;
        position: fixed;
        top: 0;
        bottom: 0;
        left: 0;
        right: 0;
        z-index: 1300;`
      : `background-color: "transparent";`}
```

How do you see it?

Reply:
Yes, that is right. It was just an example.

Reply:
Done